jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/CawsConnectorViewPanels.kt

```diff
@@ -196,6 +196,8 @@ fun cawsWizard(lifetime: Lifetime, settings: CawsSettings = CawsSettings()) = Mu
                     CawsConnectionParameters.DEV_SETTING_USE_BUNDLED_TOOLKIT to context.useBundledToolkit.toString(),
                     CawsConnectionParameters.DEV_SETTING_S3_STAGING to context.s3StagingBucket,
                     CawsConnectionParameters.DEV_SETTING_TOOLKIT_PATH to context.toolkitLocation,
+                    CawsConnectionParameters.SSO_REGION to "-1",
+                    CawsConnectionParameters.SSO_START_URL to "-1"
```

Comment:
just let these be null

Reply:
I am checking if the two connections are not the same, ie the ones sent from the URL and the one we are connected to locally, then it asks you to sign in with the required credentials. This flow is required when the connections are different in console and GW. In BID, sso_start_url and sso_region are null.
When we are trying to connect after creating a dev env in GW, the sso_region and sso_url parameters are again null. To be able to differentiate between the two, I added the value "-1". I can try changing it to the values of the current connection if that seems like a better soln

Reply:
Changing them to the current value